### PR TITLE
Login passphrase input element change 

### LINKED ISSUE #418: Change Login textarea to be a normal input
So it submits to the next step with the enter key

### PR DESCRIPTION
**Pull request by victorgdev**
## What issue does this PR close
Closes #418 

## Changes Proposed ( a list of new changes introduced by this PR)
- Changes the textarea element of the passphrase input to a regular input element.
- Adds passphrase submit on enter key pressed.

## How to test ( a list of instructions on how to test this PR)
1- Go to login page
2- Input your passphrase and press enter to go to the next step (Login Pin set up)

### CODE REVIEW

File: src/elm/Auth.elm [hunk 1]
```diff
@@ -354,7 +354,7 @@ viewLoginSteps isModal shared model loginStep =
                     ]
                 , viewFieldLabel shared.translators "auth.login.wordsMode.input" passphraseId
                 , div [ class "relative" ]
-                    [ textarea
+                    [ input
```
Comment (amiskov) [inline src/elm/Auth.elm:357]: Well, I'm not sure if it's a good change...

Here's the current behavior, all words are visible (word wrapping is enabled):
image

But with this change, we have a one-line input element without word wrapping:
image

And I'm even not sure if we can make `<input type="text">` to be multiline in HTML...

Maybe we can try to make this `input` to be multiline with some CSS/HTML properties. If not — I think it's better to have `textarea` here and use the `Enter` key handler to fire the form submit.

**Review by amiskov — CHANGES_REQUESTED** [review]

File: src/elm/Auth.elm (hunk 1 above)
Comment (heltonlr) [reply src/elm/Auth.elm:357]: I don't remember if that task asked for that.  It was better as it was, as @amiskov  commented.

**Review by lucca65 — CHANGES_REQUESTED** [review]
+1 on @amiskov comment

**Comment by lucca65** [comment]
I've removed @heltonlr review for now, lets finish the devs review first

File: src/elm/Auth.elm [hunk 2]
```diff
@@ -954,6 +955,19 @@ msgToString msg =
             [ "KeyPressed" ]
 
 
+onEnter : Msg -> Html.Attribute Msg
+onEnter msg =
```
Comment (lucca65) [inline src/elm/Auth.elm:959]: this function is already defined in a shared module: https://github.com/cambiatus/frontend/blob/cd911ed6adb67c38624abdc2823ab3b7d60e4fd8/src%2Felm%2FUtils.elm#L58

**Review by lucca65 — CHANGES_REQUESTED** [review]

**Review by lucca65 — CHANGES_REQUESTED** [review]
https://cambiatus.slack.com/archives/D016AD3RC2D/p1607176862000300

**Review by lucca65 — APPROVED** [review]

**Comment by heltonlr** [comment]
Hey @victorgdev , I clicked on "deploy/netify" to test, but I received an error.

**Comment by victorgdev** [comment]
> Hey @victorgdev , I clicked on "deploy/netify" to test, but I received an error.
> 
> "Access denied"

Hi @heltonlr, can you check again I have reset you as reviewer in this PR.

**Comment by lucca65** [comment]
@heltonlr please test this on staging! Netlify is not properly configured

**Pull request merged**